db/migrate/20230214115149_create_recipe_foods.rb

```diff
@@ -0,0 +1,11 @@
+class CreateRecipeFoods < ActiveRecord::Migration[7.0]
+  def change
+    create_table :recipe_foods do |t|
+      t.decimal :quantity
+      t.references :recipe, null: false, foreign_key: true
+      t.references :food, null: false, foreign_key: true
```

Comment:
- [ ] Lets add indexes to all foreign keys as this is a requirement for scalability